recipes/boost/all/conanfile.py

```diff
@@ -139,6 +139,9 @@ def export(self):
     @property
     def _min_compiler_version_default_cxx11(self):
         # Minimum compiler version having c++ standard >= 11
+        if self.settings.compiler == "apple-clang":
+            # For now, assume apple-clang will enable c++11 in the distant future
```

Comment:
I think 12 is 14 by default

Reply:
I meant to say "c++11 by default"

Reply:
Yep! It's our usual problem... AFAIK, in apple-clang 12 they finally set it to `c++14` by default (they skipped 11)

Reply:
According to the (failed) build on apple-clang 12, it does not have (at least) c++11 support by default.
See https://www.phoronix.com/scan.php?page=news_item&px=EXT4-Casefolding-With-Encrypt